config/newrelic.yml

```diff
@@ -29,8 +29,8 @@ common: &default_settings
     # independently. If `false`, all logging-related features are disabled.
     enabled: <%= ENV.fetch('NEW_RELIC_APPLICATION_LOGGING_ENABLED', false) %>
     forwarding:
-      # If `true`, the agent captures log records emitted by this application.
-      enabled: <%= ENV.fetch('NEW_RELIC_APPLICATION_LOGGING_FORWARDING_ENABLED', true) %>
+      # If `true`, the agent captures log records emitted by this application
```

Comment:
@kevinslin @vishnu-narayanan The Ideal way to do this is to use `ActiveModel::Type::Boolean.new.cast`